### DISCUSSION TITLE
Abnormal network reconnection

### DISCUSSION THREAD

**Discussion by XTUxiongda** [redisson/redisson#4643] [Q&A]
On Sunday afternoon, the company's network fluctuated, and the applications using Redisson began to report errors. After the network is restored, the new entry request still reports an error.The error message is as follows：
`Unable to acquire connection! Increase connection pool size and/or retryInterval settings Node source: NodeSource [slot=12606, addr=redis://192.168.10.170:7006, redisClient=null, redirect=MOVED, entry=null], command: (EVAL), params: [while true do local firstThreadId2 = redis.call('lindex', KEYS[2], 0);if firstThreadId2 == false the..., 3, fofund:lock:com.fofund.aat.service.impl.TemplateServiceImpl.edit:15805360566402580490050_01, redisson_lock_queue:{fofund:lock:com.fofund.aat.service.impl.TemplateServiceImpl.edit:15805360566402...`

redisson version is 3.11.6
redis  version is 4.0.3

I have been confused for a long time. I hope you can help me，thanks!

**Comment by mrniko** [comment]
Can you reproduce it with the latest version?

**Reply by XTUxiongda** [reply]
Unfortunately, it is difficult to reproduce。The operation and maintenance engineers are adjusting the network at the weekend.
I tried many strategies to cut off the network and close the firewall, but only occasionally.
NodeSource [slot=12606, addr=redis://192.168.10.170:7006, redisClient=null, redirect=MOVED, entry=null]
Another puzzle is that all my attempts have resulted in redirect=null in nodeSource, and MOVED has never appeared.
What will cause redirect=MOVED？

**Reply by mrniko** [reply]
This issue was fixed in 3.15.0 version
